--- virtualsmartcard-0.9/configure.ac.orig	2024-09-08 15:17:35.992591558 +0200
+++ virtualsmartcard-0.9/configure.ac	2024-09-08 15:22:52.677542595 +0200
@@ -106,7 +106,7 @@ if test "${libpcsclite}" = no ; then
 		serial wrapper driver (default to pcscd config or $(prefix)/pcsc/drivers/serial)]),
 		[serialdropdir="${enableval}"], [serialdropdir=no])
 	if test "${serialdropdir}" = no ; then
-		if test "${prefix}" = NONE ; then
+		if true ; then
 			serialdropdir="`$PKG_CONFIG libpcsclite --variable=usbdropdir`/serial"
 			if test "${serialdropdir}" = "/serial" ; then
 				case "$host" in
@@ -152,7 +152,7 @@ if test "${libpcsclite}" = no ; then
 		serial wrapper driver (default to pcscd config or /etc/reader.conf.d)]),
 		[serialconfdir="${enableval}"], [serialconfdir=no])
 	if test "${serialconfdir}" = no ; then
-		if test "${prefix}" = NONE ; then
+		if true ; then
 			serialconfdir="`$PKG_CONFIG libpcsclite --variable=serialconfdir`"
 			if test "${serialconfdir}" = "" ; then
 				case "$host" in
